lib/generators/active_record/rolify_generator.rb

```diff
@@ -82,12 +82,12 @@ def prompt_missing_user
 MSG
       end
 
-      def rails5?
-        Rails.version.start_with? '5'
+      def versioned_migrations?
+        Rails::VERSION::MAJOR.to_i >= 5
```

Comment:
In the spec below you don't use `.to_i`. I believe this is always an `Integer`? (at least it is for rails 5, I just checked)

Reply:
Good catch, I've updated this to remove the `.to_i`.